python/PiFinder/ui/object_list.py

```diff
@@ -106,6 +113,13 @@ def __init__(self, *args, **kwargs) -> None:
         self.sort_cycle = cycle(SortOrder)
         self.current_sort = next(self.sort_cycle)
 
+        if self.current_sort == SortOrder.CATALOG_SEQUENCE:
```

Comment:
shouldn't the marking menu be a class where you can call a method to enable/disable items?

Reply:
Yeah, agreed.  The developer ergonomics here are not great.  I'll rework this with the multi-level functionality as the API will need to change and I'll think a bit more about the ergonomics.